trade-finance-manager-ui/templates/utilisation-reports/_macros/fee-record-status.njk

```diff
@@ -5,7 +5,12 @@
   {% set displayStatus = params.displayStatus %}
 
   {% set tagColourClassForStatus = {
-    'TO_DO': ''
+    'TO_DO': '',
+    'MATCH': 'govuk-tag--green',
+    'DOES_NOT_MATCH': 'govuk-tag--red',
+    'READY_TO_KEY': 'govuk-tag--yellow',
+    'RECONCILED': 'govuk-tag--grey',
+    'REPORT_COMPLETED': 'govuk-tag--green'
```

Comment:
I have tested these and cross-referenced them against the miro board to make sure they match the designs